Bottom: fa0afbb920504a794165a458b584e32efebb3ce6
Top:    220121d634269cc5dec0c0d7bc1545acc1e3b8a9
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-10 17:51:33 -0700

Refresh of build-preload

---

diff --git a/Makefile.am b/Makefile.am
index 581cded..b271b45 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -24,7 +24,6 @@ src_librdmacm_la_DEPENDENCIES =  $(srcdir)/src/librdmacm.map
 
 src_librspreload_la_SOURCES = src/preload.c src/indexer.c
 src_librspreload_la_LDFLAGS = -version-info 1 -export-dynamic
-src_librspreload_la_DEPENDENCIES =  $(srcdir)/src/librspreload.map
 src_librspreload_la_LIBADD = $(top_builddir)/src/librdmacm.la
 
 bin_PROGRAMS = examples/ucmatose examples/rping examples/udaddy examples/mckey \
